### PR TITLE
info on feed import & skipping of records/image downloads

### PR DESCRIPTION
**Pull request by twessling-icas**
add some info on feed import & skipping of records/image downloads when unchanged

### CODE REVIEW

File: doc/0.120/_sources/feeds.rst.txt [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+XML file.
+
+.. warning::
+   There is still an **externalId** field in the XSD, this field is replaced by **vendorId**.
+   Please update your XML to reflect this change. This makes naming consistent between feeds and sellside API.
+   The **vendorId** field in the feeds has the same meaning and constraints as the **vendorId** field in the
+   sellside API.
+
+.. note::
+   If an ad in the feed is unchanged w.r.t. the previous time it was imported, we will skip over this ad and leave
```
Comment (homiak) [inline doc/0.120/_sources/feeds.rst.txt]: I think you can omit w.r.t - does not help to read it

File: doc/0.120/_sources/feeds.rst.txt [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+   it unchanged in our system. This also means we will **not** attempt to download the images and process them again.
```
Comment (homiak) [inline doc/0.120/_sources/feeds.rst.txt:29]: I would add - As long as the URL(s) of the image(s) for the ad have not changed

File: doc/0.120/_sources/feeds.rst.txt [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+   This will speed up processing significantly and cut down on calls to your image server.
```
Comment (homiak) [inline doc/0.120/_sources/feeds.rst.txt]: This is an optimisation that allows us to speed up....

**Review by szpakas — DISMISSED** [review]

File: doc/0.120/_sources/feeds.rst.txt [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+Below are some common asked-for scenarios and questions with their explanations/answers.
+
+- Feed file cannot be fetched
+
+When a feed file cannot be fetched, nothing will change on the user's ads.
+It's as if the import didn't happen. Since the file represents the desired
+list of ads to be live, we won't do anything if we can't get the file -
+we cannot read a change in the desired situation.
+
+- Feed file is empty
```
Comment (homiak) [inline doc/0.120/_sources/feeds.rst.txt:125]: Potentially -  ad a use-case here where images have changed but we do not see this change in ICAS

Comment (twessling-icas) [reply doc/0.120/_sources/feeds.rst.txt:125]: that's there, at the bottom of the file. Do you want some more?

File: doc/0.120/_sources/feeds.rst.txt (hunk 2 above)
Comment (twessling-icas) [reply doc/0.120/_sources/feeds.rst.txt:29]: image URLs are part of the change detection, no?

Comment (homiak) [reply doc/0.120/_sources/feeds.rst.txt:29]: yes, they are

File: doc/0.120/_sources/feeds.rst.txt (hunk 1 above)
Comment (twessling-icas) [reply doc/0.120/_sources/feeds.rst.txt]: reworded a bit

File: doc/0.120/_sources/feeds.rst.txt (hunk 4 above)
Comment (homiak) [reply doc/0.120/_sources/feeds.rst.txt:125]: Ah, I see it now sorry, missed it!

File: doc/0.120/_sources/feeds.rst.txt [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+
+.. code-block:: bash
+
+  xmllint --debug --noout --schema /path/to/admarkt1.0.xsd /path/to/yourfeed.xml
+
+For small chunks of XML you can use online validators as well, such as `<http://www.utilities-online.info/xsdvalidation/>`_ or `<https://www.freeformatter.com/xml-validator-xsd.html>`_. Note that these have a
+limit on the size of the XML you can check, but it should be more than enough to be able
+to test correctness of your structure.
+
+- I'm using the same image URLs on an ad, but the images hosted on these URLs have changed.
```
Comment (homiak) [inline doc/0.120/_sources/feeds.rst.txt]: While it does describe the problem clearly, the style of the title is out of sync with the rest. 
To be in sync with the rest it should state a problem: 
`Image updates are not in ICAS after successful feed run` 

Then you can go and explain why.

File: doc/0.120/_sources/feeds.rst.txt [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+
+If you're changing the images without changing the URLs, the changes may not be picked up,
```
Comment (homiak) [inline doc/0.120/_sources/feeds.rst.txt:166]: s/may not be picked up/ will not be picked

File: doc/0.120/_sources/feeds.rst.txt [hunk 7]
```diff
@@ -0,0 +1,169 @@
...
+in case the rest of the ad is also unchanged. We suggest adding a bogus parameter to the
```
Comment (homiak) [inline doc/0.120/_sources/feeds.rst.txt:167]: Are we sure we ignore all parameters? Perhaps, it's warrant an example

File: doc/0.120/_sources/feeds.rst.txt [hunk 8]
```diff
@@ -0,0 +1,169 @@
...
+image URL to force a re-processing of the ad and its images. Make sure to not change this
```
Comment (homiak) [inline doc/0.120/_sources/feeds.rst.txt:168]: Only do it when your image has changed and its URL has not.

File: doc/0.120/_sources/feeds.rst.txt (hunk 2 above)
Comment (twessling-icas) [reply doc/0.120/_sources/feeds.rst.txt:29]: ok, added image urls in there explicitly too

**Review by homiak — APPROVED** [review]

**Review by remenska — APPROVED** [review]

**Pull request merged**